pkg/operator/apiserver/audit/audit_policies_test.go

```diff
@@ -27,6 +36,82 @@ func TestDefaultPolicy(t *testing.T) {
 	}
 }
 
+func TestGetAuditPolicy(t *testing.T) {
+	scenarios := []struct {
+		name       string
+		goldenFile string
+		config     configv1.Audit
+	}{
+		{
+			name: "Default",
+			config: configv1.Audit{
+				Profile: "Default",
+			},
+			goldenFile: "default.yaml",
+		},
+		{
+			name: "WriteRequestBodies",
+			config: configv1.Audit{
+				Profile: "WriteRequestBodies",
+			},
+			goldenFile: "writerequestbodies.yaml",
+		},
+		{
+			name: "AllRequestBodies",
+			config: configv1.Audit{
+				Profile: "AllRequestBodies",
+			},
+			goldenFile: "allrequestbodies.yaml",
+		},
+		{
+			name: "None",
+			config: configv1.Audit{
+				Profile: "None",
+			},
+			goldenFile: "none.yaml",
+		},
+		{
+			name: "AuthenticatedOauth only",
+			config: configv1.Audit{
+				Profile: "None",
+				CustomRules: []configv1.AuditCustomRule{
+					{
+						Group:   "system:authenticated:oauth",
+						Profile: "WriteRequestBodies",
+					},
+				},
+			},
+			goldenFile: "oauth.yaml",
+		},
+		// TODO: add test with multiple customRules
+	}
+	for _, scenario := range scenarios {
+		t.Run(scenario.name, func(t *testing.T) {
+			// act
+			policy, err := GetAuditPolicy(scenario.config)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			// validate
+			if len(scenario.goldenFile) > 0 && len(scenario.goldenFile) > 0 {
```

Comment:
`if len(scenario.goldenFile) > 0 {` should be enough :)

Reply:
better twice to be sure

Reply:
fixed